pkg/apis/aws/v1alpha1/types_infrastructure.go

```diff
@@ -57,16 +57,24 @@ type Networks struct {
 	Zones []Zone `json:"zones"`
 }
 
-// Zone describes the properties of a zone
+// Zone describes the properties of a zone.
 type Zone struct {
 	// Name is the name for this zone.
 	Name string `json:"name"`
-	// Internal is  the  private subnet range to create (used for internal load balancers).
+	// Internal is the private subnet range to create (used for internal load balancers).
 	Internal string `json:"internal"`
-	// Public is the  public subnet range to create (used for bastion and load balancers).
+	// Public is the public subnet range to create (used for bastion and load balancers).
 	Public string `json:"public"`
-	// Workers is the  workers  subnet range  to create (used for the VMs).
+	// Workers is the workers subnet range to create (used for the VMs).
 	Workers string `json:"workers"`
+	// ElasticIPAllocationID contains the allocation ID of an Elastic IP that will be attached to the NAT gateway in
+	// this zone (e.g., `eipalloc-123456`). If it's not provided then a new Elastic IP will be automatically created
+	// and attached.
+	// Important: If this field is changed then the already attached Elastic IP will be disassociated from the NAT gateway
```

Comment:
```suggestion
	// Important: If this field is changed then the already attached Elastic IP will be dissociated from the NAT gateway
```
Please, add the same documentation to the internal version.